examples/imagefilter/node/main.ts

```diff
@@ -38,7 +38,7 @@ class Filters extends slint.Model<string> {
         return this.#filters[row].name;
     }
 
-    setRowData(row: number, data: string): void {
+    setRowData(row: number, data: string): boolean {
```

Comment:
Ooh? slint.Model<T>'s `setRowData` returns `void`. Why a boolean here? Is there a mismatch somewhere?

Reply:
Whoops. I was messing around and left that in by mistake. Fixed now. Not sure if this has always been the case, but on every run I see `Node.js: JavaScript Model<T>'s rowData function threw an exception` output?